docs/en/routing/basic-usage.md

```diff
@@ -0,0 +1,490 @@
+# Basic Usage
+
+## Adding Routing to an Application
+
+Begin by adding an initial route configuration, that defines a single route that maps to an identifier known as an `outlet`.
+
+>src/routes.ts
+```ts
+export default [
+	{
+		path: 'home',
+		outlet: 'home'
+	}
+];
+```
+
+Then configure the application to be "routing" aware by registering the router with an application registry.
+
+>src/main.tsx
+```tsx
+import renderer from '@dojo/framework/widget-core/vdom';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import Registry from '@dojo/framework/widget-core/Registry';
+import { registerRouterInjector } from '@dojo/framework/routing/RouterInjector';
+
+import routes from './routes';
+import App from './App';
+
+const registry = new Registry();
+// creates a router with the routes and registers the router with the registry
+registerRouterInjector(routes, registry);
+
+const r = renderer(() => <App />);
+r.mount({ registry });
+```
+
+Use the `Outlet` widget to control what widgets to render when the path for an outlet id is matched, the example will render `<div>Home</div>` for the `home` outlet.
```

Comment:
Give context where we are here? something like:
> To display something when a route is matched, we use a mechanism called an Outlet. So lets add one to show `Home` when the `home` route is visited

Reply:
Sounds good, thanks.